UndertaleModLib/Models/UndertaleGlobalInit.cs

```diff
@@ -29,7 +31,7 @@ public void Serialize(UndertaleWriter writer)
     public void Unserialize(UndertaleReader reader)
     {
         _code = new UndertaleResourceById<UndertaleCode, UndertaleChunkCODE>();
-        _code.Unserialize(reader); // TODO: reader.ReadUndertaleObject if one object starts with another one
```

Comment:
Are you sure that it's safe?

Reply:
i have not tested it, but considering that `ReadUndertaleObject(T obj)` ends up also doing `obj.Unserialize()` it seemed fine to me.

Reply:
For now this should probably be reverted unless this actually fixes something, as this may mess up object counting.